tests/functional/adapter/materialization/test_materialized_view.py

```diff
@@ -280,3 +280,48 @@ def test_set_mv_properties(self, project):
         sql = f"SHOW CREATE MATERIALIZED VIEW {catalog}.{schema}.mat_view"
         results = run_sql_with_adapter(project.adapter, sql, fetch="all")
         assert "format = 'PARQUET'" in results[0][0]
+
+
+@pytest.mark.iceberg
+@pytest.mark.skip_profile("starburst_galaxy")
+class TestIcebergMaterializedViewWithGracePeriodProperties(TestIcebergMaterializedViewBase):
```

Comment:
please rename to sth like `TestIcebergMaterializedViewWithGracePeriod` to avoid `properties`, to avoid confusion with `WITH properties` clause